### PR TITLE
Import Randovania's dol patching

### PR DESCRIPTION
**Pull request by henriquegemignani**

### CODE REVIEW

File: src/open_prime_rando/version.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # file generated by setuptools_scm
 # don't change, don't track in version control
-__version__ = version = '0.6.2.dev0'
-__version_tuple__ = version_tuple = (0, 6, 2, 'dev0')
+__version__ = version = '0.6.2.dev8'
+__version_tuple__ = version_tuple = (0, 6, 2, 'dev8')
```
Comment (duncathan) [inline src/open_prime_rando/version.py:4]: ?

**Review by duncathan — COMMENTED** [review]
it might be a good idea to split these up into folders for prime, echoes, corruption?

File: src/open_prime_rando/version.py (hunk 1 above)
Comment (henriquegemignani) [reply src/open_prime_rando/version.py:4]: Oh, forgot to update gitignore when moving things around

**Comment by henriquegemignani** [comment]
> it might be a good idea to split these up into folders for prime, echoes, corruption?

Maybe? Would cause a bunch of files with same name in different folders, but we're already doing a lot of that in RDS.

**Comment by henriquegemignani** [comment]
Things are now split

**Review by duncathan — APPROVED** [review]

**Pull request merged**